src/js/components/Video/Video.js

```diff
@@ -254,10 +274,71 @@ const Video = forwardRef(
         onClick: () => showCaptions(caption.active ? -1 : 0),
       }));
 
+      const buttonTable = {
```

Comment:
This doesn't look like a "Table". Can we call it "buttonProps"?
```suggestion
      const buttonProps = {
```